projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk-jupyterlab-extension/ui_job_properties.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2023-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+from enum import Enum
+
+
+class JobProperties(Enum):
+    name = "jobName"
```

Comment:
nit: usually, the code style naming convention of enum entries is capital letters with underscore (matching constant values)

Reply:
Done